test/test-shim.ts

```diff
@@ -70,6 +74,22 @@ test('modules already in cached do not show up in trace', t => {
   t.end();
 });
 
+test('preload traces the entire main execution and writes it to a file', t => {
+  const script = './build/test/fixtures/modA.js';
+  const tracePath = `${tmpdir()}/require-so-slow.trace`;
+  const command = `TRACE_OUTFILE=${tracePath} node -r ./build/src/index.js ${script}`;
```

Comment:
It would be good to add a test that doesn't use an env. var. to make sure the default filename is used.

Reply:
What would be the strategy for doing this that avoids writing files in the project directory? I could change working directory to a temporary one.

Reply:
:+1: